tests/schemas/builder.py

```diff
@@ -267,7 +267,10 @@ def generate_api_protocol_specs():
                 except KeyError:
                     cmd_serializer = cmd_serializers[cmd_name]
             else:
-                cmd_serializer = cmd_serializers[cmd_name]
+                try:
+                    cmd_serializer = cmd_serializers[cmd_name]
+                except:
```

Comment:
![image](https://user-images.githubusercontent.com/3187637/185403244-8a2ac75e-d66e-4a2b-a4e4-a7874032674b.png)

Reply:
What should be the proper way ?